### PR TITLE
Excel rs export fix

### PR DESCRIPTION
**Pull request by millevlada**

### CODE REVIEW

File: to.etc.domui/src/main/java/to/etc/domui/converter/ConverterRegistry.java [hunk 1]
```diff
@@ -65,6 +65,7 @@
 		register(new DomainListConverterFactory()); // Accepts anything having domain value list
 		register(new DateConverterFactory());
 		register(new LocalDateTimeConverterFactory());
+		register(new ZonedDateTimeConverterFactory());
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/converter/ConverterRegistry.java:68]: well, this got in by some side effect... that supposed to be already in skarp-prd?

File: to.etc.domui/src/main/java/to/etc/domui/converter/ZonedDateTimeConverter.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Locale;
+
+/**
+ * @author <a href="mailto:jal@etc.to">Frits Jalvingh</a>
+ * Created on 01-05-20.
+ */
+final public class ZonedDateTimeConverter implements IConverter<ZonedDateTime> {
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/converter/ZonedDateTimeConverter.java:14]: this too got in by some side effect... that supposed to be already in skarp-prd?

File: to.etc.domui/src/main/java/to/etc/domui/converter/ZonedDateTimeConverterFactory.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import to.etc.domui.component.meta.PropertyMetaModel;
+
+import java.time.ZonedDateTime;
+
+/**
+ * @author <a href="mailto:jal@etc.to">Frits Jalvingh</a>
+ * Created on 01-05-20.
+ */
+final public class ZonedDateTimeConverterFactory implements IConverterFactory {
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/converter/ZonedDateTimeConverterFactory.java:11]: same here

File: to.etc.domui/src/main/java/to/etc/domui/util/exporters/ResultSetExcelExporter.java [hunk 4]
```diff
@@ -70,7 +70,10 @@ public void writeExcel(Workbook workbook, ResultSet rs, String sheetName) {
 					}
 				}
 			}
-			sheet.createFreezePane(0, 1, 0, 2);
+			Cell cell = sheet.createRow(0).createCell(0);
```
Comment (fjalvingh) [inline to.etc.domui/src/main/java/to/etc/domui/util/exporters/ResultSetExcelExporter.java:73]: This should become optional; having this means we cannot just re-import the set?

**Review by fjalvingh — APPROVED** [review]

**Pull request merged**